Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -75,7 +75,7 @@ define([
                 return subArray;
             }
         }
-        return jsonValue;
+        return jsonValue.slice(featureId * featureSize, featureId * featureSize + featureSize);
```

Comment:
Maybe this question is more relevant to the spec, but do we expect array properties to be tightly packed like [0, 1, 2, 3, 4, 5, 6, 7, 8] or [[0, 1, 2], [3, 4, 5], [6, 7, 8]]?